docs/linux/sql-server-linux-active-directory-authentication.md

```diff
@@ -68,21 +70,12 @@ Use the following steps to join a [!INCLUDE[ssNoVersion](../includes/ssnoversion
 
    - **Ubuntu**:
 
-      Edit the `/etc/network/interfaces` file so that your AD domain controller's IP address is listed as a dns-nameserver. For example: 
-
-      ```/etc/network/interfaces
-      <...>
-      # The primary network interface
-      auto eth0
-      iface eth0 inet dhcp
-      dns-nameservers **<AD domain controller IP address>**
-      dns-search **<AD domain name>**
-      ```
+      Insure there are no dns-nameservers or dns-search entries specified in `/etc/network/interfaces` if there are, remove them and restart networking.
```

Comment:
This is incorrect. The nameservers should hold the IP of the domain controller for the domain you are joining. There can be an issue if dns-nameservers or dns-search points to the wrong domain, but these values should be specified

Reply:
The dns servers are set in resolv.conf, duplicating the setting in the ifcfg-eth file will only cause confusion and duplicate work.

Reply:
By default, the OS will auto-generate resolv.conf. On reboot, you will lose your changes. That is why we set it in the ifcfg-eth files.

Reply:
Also, you give no instrucitons on how to update /etc/resolv.conf, so even if we changed to using resolv.conf (which I don't think we should do), customers wouldn't know how to do it